src/pages/DetailsPage.js

```diff
@@ -62,14 +62,11 @@ const DetailsPage = ({personalDetails, route, translate}) => {
                 {details ? (
                     <View>
                         <View style={styles.pageWrapper}>
-                            <View
-                                style={[styles.avatarLarge, styles.mb3]}
-                            >
-                                <Avatar
-                                    style={[styles.avatarLarge]}
-                                    source={details.avatar}
-                                />
-                            </View>
+                            <Avatar
+                                containerStyle={[styles.avatarLarge, styles.mb3, styles.avatarWrapper]}
```

Comment:
Isn't the `styles.avatarWrapper` here redundant since it's already on the parent view?

Reply:
You are right. It is redundant. I removed it with latest commit.